src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -25,6 +25,7 @@
     [status-im.data-store.switcher-cards :as switcher-cards-store]
     [status-im.browser.core :as browser]
     [status-im.group-chats.core :as group-chats]
+    [status-im2.contexts.wallet.events :as wallet-2]
```

Comment:
Since there are no old wallet imports in the `status-im2.contexts.profile.login.events` ns. We can use just `wallet`.